<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SieveFilterDlg</class>
 <widget class="QDialog" name="SieveFilterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInput">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditOutput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOutput">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonApply">
        <property name="text">
         <string>小斑去除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClose">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>SieveFilterDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
